### ISSUE TITLE
Upgrade fails

### ISSUE THREAD

**Issue by spctrlth** [YunoHost-Apps/discourse_ynh#51]
Hey,
We have an upgrade that fails. It's on yunohost installed on Debian

\# yunohost --version
yunohost: 
  repo: stable
  version: 3.6.5.3
yunohost-admin: 
  repo: stable
  version: 3.6.5.1
moulinette: 
  repo: stable
  version: 3.6.4.1
ssowat: 
  repo: stable
  version: 3.6.4

Link to the errorlogs: https://matrix.org/_matrix/media/v1/download/chat.batato.be/rIOqdPwEtuwEVwpEKmEnYOzd

If you need more logs, we can get them from the server, but I assume these are the relevant parts.

**Comment by JimboJoe** [comment]
Hello, looks like the end of support of MaxMind DB, one of the Discourse dependencies (see [here](https://meta.discourse.org/t/upgrade-rebuilds-fail-due-to-maxmind-db-eol/137387/34)); so it's a Discourse upstream issue.

As they suggest, for existing installs, can you try editing /var/www/discourse/config/discourse.conf and set `refresh_maxmind_db_during_precompile_days` to 30 (instead of 2) and retry?

It won't solve new installations, though... We'll see what they'll come up with...

**Comment by spctrlth** [comment]
Hey, we retried with `refresh_maxmind_db_during_precompile_days = 30`, it didn't work. I then changed to 50 and eventually 1000, didn't work either.

The app is restored now and I was able to remove maintenance screen again, so we're not currently blocked.

Where would be the best place for us to check if we can try upgrading again? (e.g. should it work once the upstream issue is fixed or are there changes needed in this package too? And if the latter, will this issue remain open until it's fixed or is there another issue or something?)
